### PR TITLE
fix(loyalty_systems): add validity_period to abocard system #API-1017

### PR DESCRIPTION
**Pull request by aleksey-a-maltsev**
https://tillhub.atlassian.net/browse/API-1017

### CODE REVIEW

**Review by maurogri-th — APPROVED** [review]
Looks good, just not sure about the the minimum, as the minimum unit would be most likely 1.
Check it, and then do as you prefer :)

File: lib/v0/loyalty_abocard_systems/base.js [hunk 1]
```diff
@@ -68,6 +68,28 @@ const request = {
...
+        type: 'string',
+        enum: [
+          'years'
+        ]
+      },
+      value: {
+        description: 'The period value in given units',
+        example: 3,
+        type: 'number',
+        minimum: 0
```
Comment (23RoMax) [inline lib/v0/loyalty_abocard_systems/base.js:89]: Minimum 1

default 3

**Review by 23RoMax — COMMENTED** [review]
Minimum 1
Default 3

**Pull request merged**